app/src/main/res/xml/preferences_privacy.xml

```diff
@@ -3,60 +3,61 @@
 <PreferenceScreen xmlns:android="http://schemas.android.com/apk/res/android">
 
     <PreferenceCategory
-        android:title="@string/preferences_app_protection__screen_security"
-        >
+        android:title="@string/screenSecurity">
 
         <org.thoughtcrime.securesms.components.SwitchPreferenceCompat
             android:key="pref_android_screen_lock"
             android:defaultValue="false"
-            android:title="@string/preferences_app_protection__screen_lock"
-            android:summary="@string/preferences_app_protection__lock_signal_access_with_android_screen_lock_or_fingerprint" />
+            android:title="@string/lockApp"
+            android:summary="@string/lockAppDescription" />
 
         <org.thoughtcrime.securesms.components.SwitchPreferenceCompat
             android:defaultValue="@bool/screen_security_default"
             android:key="pref_screen_security"
-            android:title="@string/preferences__screen_security"
-            android:summary="@string/preferences__disable_screen_security_to_allow_screen_shots" />
+            android:title="@string/screenshotNotifications"
+            android:summary="@string/screenshotNotificationsDescription" />
 
     </PreferenceCategory>
 
     <PreferenceCategory
-        android:title="@string/preferences__message_requests_category"
-        android:key="@string/preferences__message_requests_category"
+        android:title="@string/sessionMessageRequests"
+        android:key="@string/sessionMessageRequests"
         android:persistent="false">
     </PreferenceCategory>
 
-    <PreferenceCategory android:title="@string/preferences__read_receipts">
+    <PreferenceCategory android:title="@string/readReceipts">
         <org.thoughtcrime.securesms.components.SwitchPreferenceCompat
             android:defaultValue="false"
             android:key="pref_read_receipts"
-            android:title="@string/preferences__read_receipts"
-            android:summary="@string/preferences__read_receipts_summary"/>
+            android:title="@string/readReceipts"
+            android:summary="@string/readReceiptsDescription" />
     </PreferenceCategory>
 
-    <PreferenceCategory android:title="@string/preferences__typing_indicators">
+    <PreferenceCategory android:title="@string/typingIndicators">
         <org.thoughtcrime.securesms.components.SwitchPreferenceCompat
             android:defaultValue="false"
             android:key="pref_typing_indicators"
-            android:title="@string/preferences__typing_indicators"
-            android:summary="@string/preferences__typing_indicators_summary"/>
+            android:title="@string/typingIndicators"
+            android:summary="@string/typingIndicatorsDescription" />
+        <ImagePreference />
```

Comment:
Did you mean to add this empty  image preference?

Reply:
Yes and no. We need to actually show a TypingIndicator live in the XML rather than having a string that ends with a text example of what a typing indicator looks like `(...)` - so I was going to see if I could get that working and then got distracted because strings is chaos. Thanks for the catch, I'll flag it to myself to go back to.